_posts/2021-01-13-voicemeeter-an-introduction-to-mixing-audio.md

```diff
@@ -0,0 +1,385 @@
+
+This might seem weird but,.. Lets skip regular Voicemeeter and go straight to that Banana! Yeah! Ba na na... You thought I was going to sing there? (I totally was just now), Anyways, my reason for skipping Base Voicemeeter is actually because I honestly feel like it lacks a lot of functionality that just warrants skipping it, especially since Banana is the same price of free and offers way more features and functionality. If you haven't already, this is where you would want to install Voicemeeter Banana and the extra Virtual Audio Cable(s). If you installed Voicemeeter Potato, it also installs Voicemeeter Banana. Got everything installed? Cool! Open up Voicemeeter Banana please and this is what you should see:
+
+Voicemeeter Banana Full Program Image Pre-Setup:
+{
+
+
+
+  Need Picture Here:
+
+
+
+}
+
+Something that typically gets skipped in other Voicemeeter coverage that I've seen is the ability to save settings. At any point you can save your current settings of Voicemeeter by going to Menu -> Save Settings. This saves a settings file on your computer you can then load up from just a file even on other devices should you want to transfer your settings to another device. If at any point you feel lost and want to restart the setup, you can go to Menu -> Reset Settings ( Re-Initialization) and this will make Voicemeeter appear as if you had just installed it, blank with nothing set up.
+
+Image for showing where the menu is:
+{
+
+
+
+  Need Picture here
+
+
+
+}
+
+Image for showing Save / Load / Reset Settings:
+{
+
+
+
+  Need Picture here
+
+
+
+}
+
+Voicemeeter Banana is likely flashing in the top right telling you to set up a hardware output? For good reason! you can't hear anything you are adjusting if you haven't set it up. This is ok and you should go ahead and do this. One of the Three Hardware Outputs should be your listening device (headphones, speakers, other). This is so that you wil be able to hear things once we set audio to run through voicemeeter. Another one should also be a virtual audio cable that you may have installed, it could be named such as 'VB Audio Cable'. What will we be doing with this virtual audio device? We will be sending our stream audio to it!
+
+Oh, you might have noticed that the list that pops up is repetitive with some minor variances? That is because on your computer there are different drivers which can be interacted with. In some cased WDM type drivers work better, in others MME type drivers work better. I suggest starting with WDM. If that turns out wonky you can try MME. I might cover more on the differences later but for now lets just try WDM to start! Alright, that will be all for this part, your hardware outputs should look a little something like this:
```

Comment:
The Voicemeeter manual talks about the different type of audio drivers at length. There was specific recommendations and pretty much an order of which drivers should be used in what order. Different drivers have different latency, which could be something that's relevant in timing sensitive setups like music and rhythm games.

https://vb-audio.com/Voicemeeter/Voicemeeter_UserManual.pdf

Reply:
I might have mentioned something about this in the new article revisions but if not, I could add something to it about the differences. I was just trying not to add too much extra, though a single paragraph summary might work?